File: 노누리/class2++/4153.java
package com.four;

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.Arrays;
import java.util.StringTokenizer;

public class BJ4153 {
    public static void main(String[] args) throws IOException {
        BufferedReader br=new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st;

        while(true){
            st=new StringTokenizer(br.readLine());
            int a=Integer.parseInt(st.nextToken());
            int b=Integer.parseInt(st.nextToken());
            int c=Integer.parseInt(st.nextToken());

            if(a==0 && b==0 && c==0){
                break;
            }
            int[] arr={a,b,c};
            Arrays.sort(arr);
            int max=arr[2];
            int mid=arr[1];
            int min=arr[0];

            if(max*max==(mid*mid + min*min)){
                System.out.println("right");
            }else{
                System.out.println("wrong");
            }

        }
    }
}

File: 소범기/class2/4153.py
while True:
    data = list(map(int,input().split()))
    data.sort()
    a , b , c = data[0],data[1],data[2]
    if a==0 and b==0 and c==0:
        break

    if a==0:
        print("wrong")
    elif a**2+b**2 == c**2:
        print("right")
    else:
        print("wrong")